docs/getting_started.rst

```diff
@@ -94,3 +94,159 @@ show up in Pinot.
 To show new events appearing, one can run :sql:`SELECT * FROM meetupRsvp ORDER BY mtime DESC LIMIT 50` repeatedly, which shows the
 last events that were ingested by Pinot.
 
+Experimenting with Pinot
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+Now we have a quick start Pinot cluster running locally. The below shows a step-by-step instruction on
+how to add a simple table to the Pinot system, how to upload segments, and how to query it.
+
+Suppose we have a transcript in CSV format containing students' basic info and their scores of each subject.
+
++------------+------------+-----------+-----------+-----------+-----------+
+| studentID  | firstName  | lastName  |   gender  |  subject  |   score   |
++============+============+===========+===========+===========+===========+
+|     200    |     Lucy   |   Smith   |   Female  |   Maths   |    3.8    |
++------------+------------+-----------+-----------+-----------+-----------+
+|     200    |     Lucy   |   Smith   |   Female  |  English  |    3.5    |
++------------+------------+-----------+-----------+-----------+-----------+
+|     201    |     Bob    |    King   |    Male   |   Maths   |    3.2    |
++------------+------------+-----------+-----------+-----------+-----------+
+|     202    |     Nick   |   Young   |    Male   |  Physics  |    3.6    |
++------------+------------+-----------+-----------+-----------+-----------+
+
+Firstly in order to set up a table, we need to specify the schema of this transcript.
+
+.. code-block:: none
+
+  {
+    "schemaName": "transcript",
+    "dimensionFieldSpecs": [
+      {
+        "name": "studentID",
+        "dataType": "STRING"
+      },
+      {
+        "name": "firstName",
+        "dataType": "STRING"
+      },
+      {
+        "name": "lastName",
+        "dataType": "STRING"
+      },
+      {
+        "name": "gender",
+        "dataType": "STRING"
+      },
+      {
+        "name": "subject",
+        "dataType": "STRING"
+      }
+    ],
+    "metricFieldSpecs": [
+      {
+        "name": "score",
+        "dataType": "FLOAT"
+      }
+    ]
+  }
+
+To upload the schema, we can use the command below:
+
+.. code-block:: none
+
+  $ ./pinot-distribution/target/apache-pinot-incubating-0.1.0-SNAPSHOT-bin/apache-pinot-incubating-0.1.0-SNAPSHOT-bin/bin/pinot-admin.sh AddSchema -schemaFile /Users/jlli/transcript-schema.json -exec
+  Executing command: AddSchema -controllerHost 172.25.119.20 -controllerPort 9000 -schemaFilePath /Users/jlli/transcript-schema.json -exec
```

Comment:
Please remove specific ip and host name references

Reply:
Addressed.